rclpy/src/rclpy/_rclpy_logging.c

```diff
@@ -29,19 +29,19 @@ rclpy_logging_initialize(PyObject * Py_UNUSED(self), PyObject * Py_UNUSED(args))
   Py_RETURN_NONE;
 }
 
-/// Get the global severity threshold of the logging system.
+/// Get the default severity threshold of the logging system.
 /**
  * \return severity
  */
 static PyObject *
 rclpy_logging_get_severity_threshold(PyObject * Py_UNUSED(self), PyObject * Py_UNUSED(args))
```

Comment:
Should the name of this function be changed accordingly?

Reply:
yes, there's a conceptual adaptation of the rclpy use of rcutils logging coming in another rclpy PR, but this PR is just the bare minimum to keep it compiling if https://github.com/ros2/rcutils/pull/57 gets merged (so the PRs don't block each other)

Reply:
Sounds good to me.